zerver/webhooks/freshdesk/view.py

```diff
@@ -29,7 +30,7 @@
 """.strip()
 
 
-class TicketDict(Dict[str, Any]):
+class TicketDict(WildValueDict):
```

Comment:
This class still unsafely uses `Any` and should just be removed in favor of normal dictionary lookups.